content/docs/typechecking-with-proptypes.md

```diff
@@ -1,18 +1,18 @@
 ---
 id: typechecking-with-proptypes
-title: Typechecking With PropTypes
+title: Pengecekan Tipe Dengan PropTypes
 permalink: docs/typechecking-with-proptypes.html
 redirect_from:
   - "docs/react-api.html#typechecking-with-proptypes"
 ---
 
 > Note:
 >
-> `React.PropTypes` has moved into a different package since React v15.5. Please use [the `prop-types` library instead](https://www.npmjs.com/package/prop-types).
+> `React.PropTypes` sudah dipindahkan ke paket yang berbeda sejak React v15.5. Silakan gunakan [pustaka `prop-types`](https://www.npmjs.com/package/prop-types).
 >
->We provide [a codemod script](/blog/2017/04/07/react-v15.5.0.html#migrating-from-reactproptypes) to automate the conversion.
+>Kami menyediakan [*codemod script*](/blog/2017/04/07/react-v15.5.0.html#migrating-from-reactproptypes) untuk melakukan konversi secara otomatis.
 
-As your app grows, you can catch a lot of bugs with typechecking. For some applications, you can use JavaScript extensions like [Flow](https://flow.org/) or [TypeScript](https://www.typescriptlang.org/) to typecheck your whole application. But even if you don't use those, React has some built-in typechecking abilities. To run typechecking on the props for a component, you can assign the special `propTypes` property:
+Dengan berkembangnya aplikasi kita, kit dapat menemukan banyak kesalahan atau *bugs* dengan pengecekan tipe. Untuk beberapa aplikasi, kita dapat menggunakan ekstensi JavaScript seperti [Flow](https://flow.org/) atau [TypeScript](https://www.typescriptlang.org/) untuk melakukan pengecekan tipe di aplikasi secara menyeluruh. Meskipun kamu tidak menggunakannya, React memiliki kemampuan pengecekan tipe. Untuk menjalankan pengecekan terhadap *props* disebuah komponen, kamu dapat menggunakan properti khusus `propTypes`:
```

Comment:
Typo : kit -> kita
React memiliki kemampuan pengecekan tipe -> kata built-in nya jadi hilang gak ya om?
disebuah -> di sebuah/pada sebuah/untuk sebuah

You ini mau diartikan `kita` apa `kamu`, kurang konsisten

Reply:
Makasih mas @mazipan! Gimana kalo kata-katanya jadi gini: "Meskipun kamu tidak menggunakannya, React memiliki kemampuan pengecekan tipenya sendiri." <- biar arti build-in ngga hilang.

Saya biasanya you -> kita. Tergantung gimana yg lain aja, saya ngikut.

Reply:
cc: @gedeagas @regalius dan @arfianadam, biasanya pakai kita atau kamu? biar konsisten.
Mesti disamain semua sih, biar gak bentar2 kamu, bentar2 kita.

Reply:
Built-in bisa juga diartikan "bawaan", tapi kalimat diatas udah cukup ngasih konteksnya sih.

Reply:
Hi mas @mazipan @rizafahmi , biasanya `your` kita ganti dengan `Anda` agar kesannya lebih formal 😄

Reply:
library masih boleh pake *library* mas @rizafahmi 😄 

Untuk list of agreed terms yang kita pake bisa dilihat di [Glosarium](https://github.com/reactjs/id.reactjs.org/wiki/Glosarium)

Reply:
@mazipan mungkin telat, tapi alasan you -> kita saya biasanya karena, ajakan. Di tutorial saya pake kita karena seperti membaca bareng. Kalo kamu kesannya seperti 'menyuruh'. 

@regalius kata pustaka sudah umum sih harusnya. Tapi terserah, saya ngikut